mailserver/mailserver.go

```diff
@@ -69,6 +71,21 @@ type WMailServer struct {
 
 // Init initializes mailServer.
 func (s *WMailServer) Init(shh *whisper.Whisper, config *params.WhisperConfig) error {
+	// initialize Prometheus metrics
```

Comment:
What prevents us from doing it in `mailserver/metrics.go` in `func init()?

Reply:
>In Go, the `init()` function is incredibly powerful and compared to some other languages, is a lot easier to use within your Go programs. These `init()` functions can be used within a package block and regardless of how many times that package is imported, the `init()` function will only be called once.

Oh shit, I had no idea about this. You're absolutely right, fixed.